src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeProcessorValidationTest.java

```diff
@@ -241,8 +243,8 @@ public void should_not_throw_exception_when_ocr_data_is_not_missing() throws Exc
             )
         );
 
-        Throwable throwable = catchThrowable(() ->
-            EnvelopeValidator.assertEnvelopeContainsOcrDataIfRequired(envelope)
+        Throwable throwable = catchThrowable(
+            () -> EnvelopeValidator.assertEnvelopeContainsOcrDataIfRequired(envelope)
```

Comment:
this is also unrelated change - formatting not needed. perhaps IDE auto format function is used?